django/core/validators.py

```diff
@@ -466,8 +466,8 @@ def __eq__(self, other):
 @deconstructible
 class FileExtensionValidator:
     message = _(
-        "File extension '%(extension)s' is not allowed. "
-        "Allowed extensions are: '%(allowed_extensions)s'."
+        "File extension “%(extension)s” is not allowed. "
+        "Allowed extensions are: “%(allowed_extensions)s”."
```

Comment:
I'm not sure quotes around the list of allowed extensions makes. Thoughts on dropping them?

Reply:
Agree.